backend/routes/trainer.routes.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express.Router();
+
+
+const trainerController = require('../controllers/trainer.controller');
+
+// Retrieve all trainers
+router.get('/trainers', trainerController.getTrainers);
+
+// Get single trainer
+router.get('/trainers/:trainerID', trainerController.getTrainer);
+
+//get approve status of trainer
+router.get('/trainers/:trainerID/approve', trainerController.getApproveStatus);
+
+// update approve status of trainer
+router.put('/trainers/:trainerID/approve', trainerController.updateApproveStatus);

```

Comment:
use `updateStatusOfTrainer` similarly change api path.
We try to keep our naming generic